scripts/vue-utils-v2.js

```diff
@@ -0,0 +1,55 @@
+import Vue from 'vue';
```

Comment:
why aren't these files in src, but in `scripts`? 

I suggest:

```
/util/vue/index-3.js
/util/vue/index-2.js
/util/vue/index.js
```

and with index.js having

```
export * from './index-2.js'
```

and in the script that file is replaced with

```
export * from './index-3.js'
```

Reply:
yeah that's cleaner. thanks! https://github.com/algolia/vue-instantsearch/pull/1009/commits/c3eef175996ab58ab52b3de554f0d803bf10092a